test/verify/check-storage-lvm2

```diff
@@ -21,6 +21,7 @@ import parent
 from storagelib import *
 from testlib import *
 
+@nondestructive
```

Comment:
These tests modify `/etc/fstab`.  They also clean it up, but only when they run all the way to the end.  If a test fails in the middle, we might have something in `/etc/fstab` that will cause the next boot to fail.

Reply:
I guess we should backup/restore `/etc/fstab` and `/etc/crypttab` around all the storage tests.

Reply:
Good idea, done in a separate commit.